### PR TITLE
SF-1762 - Added job clone functionality

### PR DESCRIPTION
**Pull request by krispeterson**

### CODE REVIEW

File: src/commands/jobs/delete.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { runCommand } from '../../util/command';
 
 export = {
-	command: 'delete <id>',
+	command: 'delete [id]',
```
Comment (krispeterson) [inline src/commands/jobs/delete.ts]: This was changed only for consistency, as was `show.ts` below. What is our standard? `[]` or `<>`?

Comment (smbecker) [reply src/commands/jobs/delete.ts]: It [looks like](https://github.com/yargs/yargs/blob/master/docs/examples.md#yargs-is-here-to-help-you) `yargs` uses `<>` for command arguments and `[]` for option values.

Comment (krispeterson) [reply src/commands/jobs/delete.ts]: Perfect. Those two changes should be valid then.

Comment (smbecker) [reply src/commands/jobs/delete.ts]: Actually, based on that pattern, it should be `delete <id>` rather than `delete [id]`.

Comment (krispeterson) [reply src/commands/jobs/delete.ts]: Updated this and others to use `<>` for command arguments. However, I did this only for those arguments which were required. For those arguments which appear optional, I left them qualified with `[]`. That may be splitting hairs, but that makes sense to me.

**Pull request merged**